scripts/run_yamlcheck.sh

```diff
@@ -1,4 +1,9 @@
 #!/bin/bash -xe
+
+arch=$(uname -m)
+echo arch
+if [ "$arch" = "x86_64" ];
+then
 docker run -t -v `pwd`/deploy/kubernetes/v1.13:/deploy/kubernetes/v1.13 garethr/kubeval deploy/kubernetes/v1.13/*.yaml
 docker run -t -v `pwd`/deploy/kubernetes/v1.14:/deploy/kubernetes/v1.14 garethr/kubeval deploy/kubernetes/v1.14/*.yaml
-
+fi
```

Comment:
add `else` echo "skipping k8s yaml validations since it only supports x86" (something like this)

Reply:
done